changelogs/fragments/138-docker_container-allow-memory-swap-unlimited.yml

```
@@ -0,0 +1,2 @@
+bugfixes:
```

This is more a feature, not a bugfix:
```suggestion
minor_changes:
```